Checkpoint successfully loaded from checkpoints/BalanceLoss/enet_checkpoint_epoch_96.pth (Epoch 96)
length of val41
 and <torch.utils.data.dataloader.DataLoader object at 0x7fda70828070>
Traceback (most recent call last):
  File "eval.py", line 96, in <module>
    evaluate()
  File "eval.py", line 66, in evaluate
    combined_row = visualize_first_prediction(
TypeError: visualize_first_prediction() missing 1 required positional argument: 'instance_labels'
